close #26: デザインが崩れている問題を修正・一部スタイルの変更

kazuemon opened the pull request:
## 📝 関連issue / Related Issues

- close #26 

## ⛏ 変更内容 / Details of Changes

### 各アイテムのタイトル部分が改行されることで崩れてしまう問題を修正

#### 変更前
![image](https://user-images.githubusercontent.com/12812934/97688369-66386a80-1add-11eb-8bf8-eebbdbe0852d.png)

#### 変更後
![image](https://user-images.githubusercontent.com/12812934/97688439-7a7c6780-1add-11eb-8fd8-b49bcfa74127.png)

-----

### CSS Gridを用いてスタイルをリファクタ

同じ行のアイテムの高さは揃えられますが、その際、タイトルの縦中央揃えはされません。
IE用のprefixが必要であれば...あとから足すことも検討しています。

#### 変更前
![image](https://user-images.githubusercontent.com/12812934/97687632-fd50f280-1adc-11eb-9157-db84dd06c122.png)

#### 変更後
![image](https://user-images.githubusercontent.com/12812934/97687516-f32ef400-1adc-11eb-8a33-ae2f92811946.png)


## 📸 スクリーンショット / Screenshots

見た目には大きな差異はありません。

### 変更前
![image](https://user-images.githubusercontent.com/12812934/97685818-7dc32380-1adc-11eb-807e-ae634e68d702.png)

### 変更後
![image](https://user-images.githubusercontent.com/12812934/97685517-697f2680-1adc-11eb-9ed0-4645b06b3fbf.png)

Commit:
fix, refactor: アイテムの表示ずれを修正, 一部コードをリファクタ

Commit:
change, clean: ファイル名のtypoを修正, 不要なスタイルを削除

kazuemon commented:
コード比較で確認した限り、リンクやサムネイルのミスはないと思われますが、念のためそちらもご確認をお願い致します。

Commit:
clean : インデントを修正

07JP27 commented on index.html:
```diff
@@ -60,459 +60,459 @@ <h2 class="category-header">このサイトについて</h2>
 
 			<!-- シェアボタンエリア -->
-			<div　class="col s12 m12 l12 xl12">
+			<div class="col s12 m12 l12 xl12" style="margin: 4rem 0;">
```
スタイルはインラインではなくCSSファイルにまとめたいのですが可能でしょうか。

kazuemon replied:
了解しました、対応します。

Commit:
update: スタイルを一部修正

- 記述順序を変更
- インデントを修正
- 使用されていないスタイルを削除
- ヘッダーロゴのpadding-leftをハンバーガーメニューに変わる幅で無効化
- .title-container をflexboxを使用した形に変更
- シェアボタンエリアのタグを修正、クラス化してインラインCSSを除去

kazuemon commented:
@07JP27 当該インラインCSSの除去も含め、さらにいくつか変更いたしました。お手数をおかけしますが、ご確認をお願いいたします。

Commit:
update: ナビバーの挙動を変更

- ロゴを押した際に一番上に戻るように
- ハンバーガーメニューを押したときに一番上に戻らないように

07JP27 reviewed (approved):
PRありがとうございました。
本サイトは自動デプロイではないため一定サイクルでリリースします。
本変更は次回更新時にサイトに適用されます。

kazuemon commented:
レビュー・マージありがとうございます。
了解いたしました。
